dev/hal/inc/SdCard.h

```diff
@@ -0,0 +1,61 @@
+/**
+ * @file SdCard.h
+ * @author noura36
+ * @brief
+ * @version 0.1
+ * @date 2024-07-28
+ *
+ * @copyright Copyright (c) 2024
+ *
+ */
+#ifndef DEV_HAL_INC_SDCARD_H_
+#define DEV_HAL_INC_SDCARD_H_
+
+using namespace stm32::dev::mcal::pin; 
+using namespace stm32::dev::mcal::gpio;
+using namespace stm32::dev::mcal::spi;
+#define SD_TIMEOUT    (200)
+namespace stm32 {
+namespace dev {
+namespace hal {
+namespace sdcard {
+enum  SDCommand {
+    CMD0 = 0,     // GO_IDLE_STATE - Reset card to idle state
```

Comment:
I don't think its not difficult to follow enum coding style. Also repeating the same comment is an indication for a lot of bad things, so please avoid this